docs/docs/guides/automate/sensors/testing-sensors.md

```diff
@@ -63,6 +63,7 @@ To unit test sensors, you can directly invoke the sensor's Python function. This
   path="docs_snippets/docs_snippets/concepts/partitions_schedules_sensors/sensors/sensors.py"
```

Comment:
Not really part of this PR, but the bold text in the tabs that repeats the tab name is pretty distracting, so feel free to remove it